README.md

```diff
@@ -388,3 +388,19 @@ specifically disclaims any liability with respect to this document
 and no contractual obligations are formed either directly or
 indirectly by this document. The technologies, functionality, services,
 and processes described herein are subject to change without notice.
+
+## **Important Notice: This Repository is Deprecated**
+
+**Important Notice**
+
+This repository (fsxa-pattern-library) has been deprecated and is no longer actively maintained. Its usage comes with potential risks and limitations. We recommend users to consider alternative solutions for their needs.
+
+**Alternative Solution:**
+
+The functionality and features of this repository have been transitioned to the `content-api` repository, which is the new replacement for FSXA stack. You can find the latest updates, improvements, and ongoing support there.
```

Comment:
You say it was transitioned to the `content-api` repository. Is that actually correct? Because my understanding is that the `fsxa-api` was simply renamed to the content api library and the actual replacement for the pattern-library, nuxt-module, etc is the [crownpeak-pwa-template](https://github.com/e-Spirit/crownpeak-pwa-template). Am I missing something?